.env.template

```diff
@@ -3,4 +3,10 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
+
 SECRET_KEY=secret-key
+
+TRELLO_BOARD_ID=<TRELLO_BOARD_ID>
+
+TRELLO_KEY=<TRELLO_KEY>
+TRELLO_TOKEN=<TRELLO_TOKEN>
```

Comment:
Thanks for keeping this updated, makes my job testing the code much easier!

Reply:
Even better when my API key isn't included in the file too!